src/python/pants/backend/docker/target_types.py

```diff
@@ -206,6 +206,25 @@ def option_values(self) -> Iterator[str]:
             yield f"id={secret},src={os.path.normpath(full_path)}"
 
 
+class DockerBuildSSHOptionField(DockerBuildOptionFieldMixin, StringSequenceField):
+    alias = "ssh"
+    default = ()
+    help = (
+        "SSH agent socket or keys to expose to the build (only if BuildKit enabled) "
+        "(format: default|<id>[=<socket>|<key>[,<key>]])\n\n"
```

Comment:
Thanks for specifying the grammar! Perhaps an example would help too in a followup paragraph? I appreciate the example with `RUN --mount=type=ssh`.

Reply:
Not blocking. My brain works better seeing examples, but the help message is already quite excellent :)

Reply:
Yeah, the lack of example here is mainly due to me not having a clue! :P :D 

Never used this option myself, so it's just copy paste from the docker docs. ;)

Reply:
I'll land this as-is for now. we can always come back and improve on it later..